packages/front-end/components/Experiment/Results.tsx

```diff
@@ -194,6 +195,20 @@ const Results: FC<{
                 dimension,
               }),
             });
+            trackSnapshot("create", {
+              source: "VariationIdWarning",
+              experiment: experiment.id,
+              engine: statsEngine,
+              regressionAdjustmentEnabled: !!regressionAdjustmentEnabled,
+              sequentialTestingEnabled: !!experiment.sequentialTestingEnabled,
+              sequentialTestingTuningParameter:
+                experiment.sequentialTestingTuningParameter || null,
```

Comment:
Is this allowed to be 0? If so, 0 will not happen because JavaScript. 

```js
> var myValue = 0 || null
undefined
> myValue
null
```

Reply:
if you need to support 0 you can do `typeof experiment.sequentialTestingTuningParameter === 'number' ? experiment.sequentialTestingTuningParameter : null`

Reply:
It shouldn't be 0, but I prefer being more explicit so I'll implement something more explicit as you suggest.

Reply:
I'm just going to allow the field to be undefined which is fine for our tracking/data purposes.